cmake/projects/libsodium/hunter.cmake

```diff
@@ -17,10 +18,14 @@ hunter_add_version(
     00a999629f1f8f500cef29818135f8f889f44e41
 )
 
+hunter_configuration_types(libsodium CONFIGURATION_TYPES Release)
 if(MSVC)
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_msbuild)
 else()
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_autogen_autotools)
```

Comment:
Its a C library only, I see that the `CFLAGS` are being passed and being used. C++ flags under `CXXFLAGS` are not used. What kind of flags were you expecting? Maybe my test is wrong :|